sonar-helm-for-iac/main_test.go

```diff
@@ -276,3 +276,43 @@ func Test_to_protobuf_invalid(t *testing.T) {
 	assert.Equal(t, "", templateFromProto.Template)
 	assert.Equal(t, "template: a.yaml:1: unclosed action", templateFromProto.Error)
 }
+
+func Test_evaluate_template_default_function(t *testing.T) {
```

Comment:
Not sure if we need this test, but at the beginning it was not working for me in java, so I tested it internally in go code first.

Reply:
It's ok to have extra test here